docs/program/apis/disconnect.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Session Connectivity Resource Disonnection during a Session"
+linkTitle: "Session Connectivity"
+weight: 20
+type: "docs"
+description: "How to troubleshoot a resource losing connection during a robot session."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session"]
+---
+
+A guide to what happens when your robot loses its network connection during a session.
+
+### How does `viam-server` work if you lose connectivity during a session?
+
+If you lose connectivity, `viam-server` should stop this robot's session, therefore stopping the flow of power to all actuating resources.
+
+### How do Viam's SDKs work if you lose connectivity during a session?
+
+As a client of your robot, the Python or TypeScript SDKs will have their connection to the robot stopped when the robot loses connectivity, if the session is ended properly.
```

Comment:
This is kind of too vague w/ "ended properly" but i'm not sure how to explain it and not make redundant info from the other sessions page. @dannenberg do you have any suggestions here if this isn't phrased super weirdly or wrong?